README.md

```diff
@@ -22,6 +22,37 @@ Alternatively, you can clone the package from GitHub and set everything up with
 Please note this package requires Tensorflow for Deepbooru tagging and other required packages which do take up quite a lot of disk space (>400MB).
 A future release of this package will offer an option without the need to install Tensorflow if you don't want to use Deepbooru tagging.
 
+### Docker Instructions
+If you would like to run the toolkit in a Docker container instead, follow the
+instructions below.
+
+1. Copy `docker-compose.yml` to the location where you want to run the toolkit.
+1. Copy `config_sample.toml` to the same location, renaming to `config.toml` and
+replacing with your configuration.
+1. Copy `crontab_sample` to the same location, renaming to `crontab` and adding
+   the commands you would like to run regularly. An example command is provided
+   in `crontab_sample`.
+1. If you would like to use deepbooru or tag files, create `misc/deepbooru`
+   and/or `misc/tags` in the same location and follow the instructions linked
+   below
```

Comment:
Another bind volume is necessary for the `src_path` in the `config.toml`, otherwise, the script won't start.

Something like:

> 1. Edit the `docker-compose.yml` file and add another entry under `volumes` for your `src_path` specified in `config.toml`. It needs to be in the following format: `- <local_src_path>:<target_path>`. It's recommended to specify the target path as `/szurubooru-toolkit/upload_src`. In that case, the `src_path` in your `config.toml` should point to `/szurubooru-toolkit/upload_src`.

Reply:
ooh good point, I'll replace the temp mapping with that